### PR TITLE
feat: Add new operator mapArrayReverse

### LINKED ISSUE #13: Implement `mapArrayReverse`

### PR DESCRIPTION
**Pull request by web-dave**
this was a easy one.
This should fix #13

### CODE REVIEW

File: src/operators/map-array-reverse.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { map } from 'rxjs/operators';
+
+/**
+ * Map values of an array.
+ *
+ * - - - - [1, 2, 3, 4, 5] - - - -
+ *       mapArrayReverse()
+ * - - - - [5, 4, 3, 2, 1] - - - -
+ *
+ * @param fn Predicate function to apply to each item in the array.
```
Comment (Brocco) [inline src/operators/map-array-reverse.ts]: Please remove this param from the JS Docs, this operator does not accept any params.

File: README.md [hunk 2]
```diff
@@ -36,3 +36,14 @@ of([1, 2, 3, 4])
...
 // 10
 ```
+
+### mapArrayReverse
+
+Reverses an array in an Observable.
+
+```js
+of([1, 2, 3, 4, 5])
+  .pipe(mapArrayReverse()
```
Comment (Brocco) [inline README.md]: Missing a close paren, which will close the call to `pipe`

File: src/operators/map-array-reverse.ts (hunk 1 above)
Comment (web-dave) [reply src/operators/map-array-reverse.ts]: Oh nooo,  I'll fix it

**Comment by web-dave** [comment]
Fixed it and thx for being so nice

**Pull request merged**